modules/querying/pages/query-operations.adoc

```diff
@@ -760,24 +760,26 @@ A vertex ID would be `"Tom,456"`, consisting of the `name` attribute and the `id
 |Set or bag of primitives |Use square brackets to enclose the collection
 of values. |A set of integers: `[1,5,10]`
 
-|`VERTEX<type>` |If the vertex type is specified in the query
+|Typed vertex parameter: `VERTEX<type>` |If the vertex type is specified in the query
 definition, then the vertex argument is `**vertex_id**` |The vertex type
 is `person` and the desired ID is `person2`. `"person2"`
 
-|`VERTEX`(type not pre-specified) |If the type is not defined in the
-query definition, then the argument must provide both the id and type in
-parentheses:*(vertex_id, vertex_type)* |A vertex with ID `"person1"` and
-`type="person`: `("person1","person")`
+|Generic vertex parameter: `VERTEX` |If the type is not defined in the
+query definition, then the argument must be a tuple of strings with both the id and type: `("vertex_id", "vertex_type")` |A vertex with ID `"person1"` and
+`type="person"`: `("person1","person")`
```

Comment:
Maybe capitalize the vertex type to be Person. This makes it a little easier to distinguish person1 from Person, and it follows our GSQL Style Guide.